WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -143,6 +147,10 @@ class MySiteViewModel
     val onNavigation = merge(_onNavigation, siteStoriesHandler.onNavigation)
     val onMediaUpload = _onMediaUpload as LiveData<Event<MediaModel>>
     val onUploadedItem = siteIconUploadHandler.onUploadedItem
+    val onExternalQuickStartFocusPointVisibilityChange = quickStartRepository.activeTask
```

Comment:
What do you think about moving this to the Main Activity view model and observing the active task directly there? I think it makes sense to do it there since the focus point is shown on that level and not in the MySite fragment/viewmodel.

Reply:
Thanks for the suggestion, it makes total sense. I have now moved the logic to the `MainActivity` level. 

Since I have also moved the logic to add/remove a focus point view from the `ImprovedMySiteFragment` to the `MainActivity`, it seemed to make sense to reuse that for the `PUBLISH_POST` task as well (the logic on the view model remains the same). I've tested this with the flag both on and off and it seems to be working well. Let me know what you think.